dual_net.py

```diff
@@ -238,7 +255,11 @@ def my_res_layer(inputs):
 
 
 def get_estimator(working_dir):
-    return tf.estimator.Estimator(model_fn, model_dir=working_dir)
+    run_config = tf.estimator.RunConfig(save_summary_steps=500)
```

Comment:
This reduces the logging by 5x (but update ratios still represent 90% of the log)